docs/src/Callout.doc.js

```diff
@@ -22,168 +47,346 @@ card(
+
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).
+        `,
         href: '',
       },
       {
         name: 'secondaryAction',
         type:
           '{| accessibilityLabel?: string , href?: string, label: string, onClick?: ({ event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement | SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> }) => void |}, onNavigationOptions: ({ [string]: Node | ({| +event: SyntheticEvent<> |}) => void }) => void, rel: "none" | "nofollow", target: "null" | "self" | "blank" |}',
-        required: false,
         defaultValue: null,
-        description: [
-          'Button to render inside the callout as the secondary call-to-action to the user. The label text has a fixed size.',
-          '- href: If href is supplied, the action will serve as a link, while if no href is supplied, the action will be a button',
-          '- label: Text to render inside the button to convey the function and purpose of the button. The button text has a fixed size.',
-          '- accessibilityLabel: Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
-          '- onClick: Callback fired when the button component is clicked (pressed and released) with a mouse or keyboard.',
-          `- onNavigationOptions: onNavigationOptions works in conjunction with a Provider. Pass custom props to onNavigation. See Provider for examples. onNavigation's type is flexible. Each key's value is a React.Node or an event handler function. Optional with href.`,
-          'Accessibility: `accessibilityLabel` populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the button `text`.',
-        ],
+        description: `
+          Secondary action for people to take on Callout. If \`href\` is supplied, the action will serve as a link. If no \`href\` is supplied, the action will be a button.
+
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).
+        `,
         href: '',
       },
       {
         name: 'type',
         type: `"error" | "info" | "warning"`,
         required: true,
         defaultValue: null,
-        description: [
-          'Select a type of callout:',
-          '-`error`: To provide error messaging to the user.',
-          '-`info`: To provide general information to the user.',
-          '-`warning`: To provide a warn message to the user.',
-        ],
-        href: '',
+        description: `
+        The type of Callout. See the [Variants](#Variants) to learn more.
```

Comment:
the word `type` might be redundant. `Callout categories.` I was thinking that this is like traffic signs, each style conveys different categories, different information messages.